packages/frontend-shared/src/graphql/urqlClient.ts

```diff
@@ -41,6 +41,7 @@ declare global {
     __CYPRESS_MODE__: 'run' | 'open'
     __RUN_MODE_SPECS__: SpecFile[]
     __CYPRESS_TESTING_TYPE__: 'e2e' | 'component'
+    __CYPRESS_BROWSER__: Partial<Browser> & {majorVersion: string | number}
```

Comment:
Didn't we already have a Cypress.platform and Cypress.browser. Why do we need this extra variable?

Reply:
Good point, I wasn't aware of Cypress.browser, I'll take a look and if that works, I'll get rid of this global

Reply:
I did implement this, `Cypress` is not available right away when the setup function runs, we have to wait for it, which leads to polling for it. I'm going to leave it as is, I think it's a little more simple & reliable this way, plus header can render without even the tiny few ms extra delay.

Reply:
OK, good to know